gpjax/abstractions.py

```diff
@@ -175,6 +178,7 @@ def fit_batches(
     batch_size: int,
     n_iters: Optional[int] = 100,
     log_rate: Optional[int] = 10,
+    verbose: Optional[bool] = True,
```

Comment:
Do you think defaulting to True is sensible? I can see benefits for defaulting to False or True, but the former feels less invasive? Keen to get your thoughts though.